networkx/algorithms/approximation/tests/test_clique.py

```diff
@@ -22,21 +22,21 @@
 
 
 def is_independent_set(G, nodes):
-    """Returns ``True`` if and only if ``nodes`` is a clique in ``G``.
+    """Returns True if and only if `nodes` is a clique in `G`.
```

Comment:
That's the numpydoc/napoleon standard (at least as far as [their example docs](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/sphinxcontrib.napoleon.html) show). That's true for True, False and None.  We don't have to follow those guidelines all the time, so if it's unclear whether you mean the python object or the concept you'll use some sort of formatting. 

I can't find a good example for formatting the type int though in a paragraph.  It seems that the examples use int without formatting, but the type of a parameter is automatically italicized. Since it is not even an english word I think it is Ok to go without formatting for types too.